src/platform/i386/exception.c

```diff
@@ -177,11 +176,11 @@ machine_check_abort_handler(struct pt_regs *regs)
 }
 
 int
-smid_float_pt_except_fault_handler(struct pt_regs *regs)
+simd_float_pt_except_fault_handler(struct pt_regs *regs) //it is defined as simd at the beginning, does not matter
 {
 	print_regs_state(regs);
-	die("FAULT: SMID Floating-point Exception\n");
-
+	die("FAULT: SIMD Floating-point Exception\n");
```

Comment:
Does this exception ever fire?

Reply:
This exception has never been fired, but there was a typo, SMID should be SIMD.